generators/common/templates/README.md.ejs

```diff
@@ -282,7 +282,7 @@ docker-compose -f src/main/docker/sonar.yml up -d
 Then, run a Sonar analysis:
 
 ```
-<% if (buildTool === 'maven') { %>./mvnw -Pprod clean test sonar:sonar<% } else { %>./gradlew -Pprod clean test sonarqube<% } %>
+<% if (buildTool === 'maven') { %>./mvnw -Pprod clean verify sonar:sonar<% } else { %>./gradlew -Pprod clean test sonarqube<% } %>
```

Comment:
Forgot to change it here, but for gradle we should use `./gradlew -Pprod clean check sonarqube`

Reply:
> Shall I take a look in the gradle part?

Would you like to do the gradle part or the the review only ?